proposals/gc/MVP.md

```diff
@@ -139,33 +179,35 @@ In addition to the [existing rules](https://github.com/WebAssembly/function-refe
 
 #### Runtime Types
 
-* Runtime types (RTTs) are explicit values representing types at runtime; a value of type `rtt <n> <heaptype>` is a dynamic representative of static type `<heaptype>`.
+* Runtime types (RTTs) are explicit values representing concrete types at runtime; a value of type `rtt <n>? <typeidx>` is a dynamic representative of the static type `<typeidx>`.
```

Comment:
This needs an update to the binary encoding section. The decoder needs a way to know whether the `<n>` is present or not. One possible solution is to have different encodings for rtt-with-n and rtt-without-n. Another possible solution is to use a sentinel value for absent `<n>`, such as -1. I'm sure there are other possible solutions too.
(The binary encoding section should also be updated to reflect the heaptype->typeidx change.)

Reply:
Ah, indeed. I added opcode `-0x18` for rtt without n.